extensions/amp-story-auto-ads/0.1/utils.js

```diff
@@ -46,11 +46,20 @@ export function getUniqueId(win) {
  * @param {Document} doc
  * @return {!IArrayLike}
  */
-export function getA4AMetaTags(doc) {
+export function getA4AVarsMetaTags(doc) {
   const selector = 'meta[name^=amp4ads-vars-]';
   return doc.querySelectorAll(selector);
 }
 
+/**
+ * Finds all story ad meta tags starting with `amp-cta-`.
+ * @param {Document} doc
+ * @return {!IArrayLike}
+ */
+export function getAmpCtaMetaTags(doc) {
+  return doc.querySelectorAll('meta[name^=amp-cta-]');
```

Comment:
did you try one query to select both?
`meta[name^=amp-cta-],meta[name^=amp4ads-vars-]`

Reply:
Switched to single selector.